events/class_tests/meeting_tests.py

```diff
@@ -30,12 +30,20 @@ def persist_possible_meeting(possible_meeting0):
 
 
 def get_event_participant_from_db():
-    return EventParticipant.objects.filter(is_creator=True)[0]
+    return EventParticipant.objects.filter(is_creator=True).first()
 
 
 @pytest.mark.django_db
 class TestMeeting():
 
+    @pytest.fixture
+    def expected_meeting_results(self):
+        return [
+            OptionalMeetingDates.objects.get(id=3),
+            OptionalMeetingDates.objects.get(id=4),
+            OptionalMeetingDates.objects.get(id=5),
+        ]
```

Comment:
more pythonic way to do it like:
```suggestion
        return [ OptionalMeetingDates.objects.get(id=f"{id}") for id in range(3, 6) ]
        same as:
        return [ OptionalMeetingDates.objects.get(id=f"{id}") for id in [3, 4, 5] ]

```

Reply:
done. thanks 😄